src/components/dashboard/Dashboard.js

```diff
@@ -100,7 +100,7 @@ const Dashboard = props => {
   const [balanceBlockWidth, setBalanceBlockWidth] = useState(70)
   const [showBalance, setShowBalance] = useState(false)
   const [headerContentWidth, setHeaderContentWidth] = useState(initialHeaderContentWidth)
-  const [headerHeightAnimValue] = useState(new Animated.Value(210))
+  const [headerHeightAnimValue] = useState(new Animated.Value(165))
```

Comment:
the header height in the design is 210

Reply:
I couldn't find that exact dimension on the design link provided in the issue, but seems to be closer to 165 in there (this height is just for the wrapper for the avatar, username & balance).
Where can I find that 210, is there an updated design file?